Adding docs about Grafana upgrade

dockerymick opened the pull request:
https://issues.redhat.com/browse/ACM-4118

dockerymick committed:
Adding docs about Grafana upgrade

oafischer commented on observability/verifying_observability.adoc:
```diff
@@ -0,0 +1,16 @@
+[#verifying-observability-version]
+= Verifying observability version
+
+Verify the observability version from the command line interface (CLI) or from Grafana user interface. 
```
```suggestion
Verify the observability version from the command line interface (CLI) or from the Grafana user interface. 
```

oafischer commented on observability/verifying_observability.adoc:
```diff
@@ -0,0 +1,16 @@
+
+The Grafana version that is currently deployed within the cluster environment is displayed.
+
+Alternatively, you can navigate to the _Manage_ tab in the Grafana dashboard. Scroll to the end of the page, where the version is listed
```
```suggestion
Alternatively, you can navigate to the _Manage_ tab in the Grafana dashboard. Scroll to the end of the page, where the version is listed.
```

oafischer reviewed (approved):
Looks good, just some small typos.

dockerymick committed:
Update observability/verifying_observability.adoc

accepting suggestion

Co-authored-by: Oliver Fischer <108426932+oafischer@users.noreply.github.com>

dockerymick committed:
Update observability/verifying_observability.adoc

Great catch, accepting suggestion

Co-authored-by: Oliver Fischer <108426932+oafischer@users.noreply.github.com>

dislbenn commented on observability/verifying_observability.adoc:
```diff
@@ -0,0 +1,16 @@
+[#verifying-observability-version]
+= Verifying observability version
+
+Verify the observability version from the command line interface (CLI) or from the Grafana user interface. 
+
```
> Verify the observability version from the command line interface (CLI) or from the Grafana user interface.

From the Grafana user interface, you are only able to verify the Grafana version, not the Observability version. Similar to Thanos, if you execute the command `thanos --version` within the following pods, you will be able to see what Thanos version that Observability is deploying:
- observability-thanos-compact
- observability-thanos-query
- observability-thanos-query-frontend
- observability-thanos-receive-default
- observability-thanos-rule
- observability-thanos-store-shard

dislbenn replied:
@dockerymick Can we add the Thanos version verification step to this section also?

dockerymick committed:
Update design_grafana.adoc

Moved verifying Grafana version to the Grafana docs

dockerymick committed:
Update observability_enable.adoc

Adding section to verify Thanos version

dockerymick committed:
Update whats_new.adoc

dockerymick committed:
Delete verifying_observability.adoc

Removing file

dockerymick committed:
Merge branch '2.8_stage' into mj-grafana-ACM-4118

dockerymick commented:
@dislbenn thanks for the feedback. I made a couple of updates. When you have an opportunity, will you rereview the PR?

dislbenn reviewed (approved):
The changes look good to me! Thanks @dockerymick

dockerymick committed:
Update customize_observability.adoc

Removing comment bc issue was created: https://issues.redhat.com/browse/ACM-4457

dockerymick commented:
Thanks for your review ! I will merge after I run the link check tool
